assets/syntaxes/02_Extra/sparql.sublime-syntax

```diff
@@ -0,0 +1,140 @@
+%YAML 1.2
+---
+# Based on https://github.com/stardog-union/stardog-vsc/tree/0ce931c26529fdaf5c6dd58d91e589bbd6944a1d/stardog-rdf-grammars/syntaxes
+name: SPARQL
+file_extensions:
+  - rq
+  - sparql
+scope: source.sparql
+contexts:
+  main:
+    - include: rule-constraint
+    - include: iriref
+    - include: prefix
+    - include: prefixed-name
+    - include: comment
+    - include: special-predicate
+    - include: literals
+    - include: language-tag
+    - include: query-keyword-operators
+    - include: functions
+    - include: variables
+    - include: expression-operators
+  comment:
+    - match: "#.*$"
+      scope: comment.line.number-sign.turtle
+  boolean:
+    - match: \b(?i:true|false)\b
+      scope: constant.language.sparql
+  expression-operators:
+    - match: (?:\|\||&&|=|!=|<|>|<=|>=|\*|/|\+|-|\||\^|\?|\!)
+      scope: support.class.sparql
+  functions:
+    - match: \b(?i:concat|regex|asc|desc|bound|isiri|isuri|isblank|isliteral|isnumeric|str|lang|datatype|sameterm|langmatches|avg|count|group_concat|separator|max|min|sample|sum|iri|uri|bnode|strdt|uuid|struuid|strlang|strlen|substr|ucase|lcase|strstarts|strends|contains|strbefore|strafter|encode_for_uri|replace|abs|round|ceil|floor|rand|now|year|month|day|hours|minutes|seconds|timezone|tz|md5|sha1|sha256|sha384|sha512|coalesce|if)\b
+      scope: support.function.sparql
+  integer:
+    - match: '[+-]?(?:\d+|[0-9]+\.[0-9]*|\.[0-9]+(?:[eE][+-]?\d+)?)'
+      scope: constant.numeric.turtle
+  iriref:
+    - match: '<[^\x20-\x20<>"{}|^`\\]*>'
+      scope: entity.name.type.iriref.turtle
+  language-tag:
+    - match: '@(\w+)'
+      scope: meta.string-literal-language-tag.turtle
+      captures:
+        "1": entity.name.class.turtle
```

Comment:
Note: Sublime Text would fail to load this syntax definition, due to the quoting on the capture group numbers in various places.